### PR TITLE
[MLIR][Refactor] Rename b (OpBuilder) to builder

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**

### CODE REVIEW

File: src/libasr/codegen/asr_to_mlir.cpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ class ASRToMLIRVisitor : public ASR::BaseVisitor<ASRToMLIRVisitor>
     std::string src;
 
     std::unique_ptr<mlir::MLIRContext> context;
-    std::unique_ptr<mlir::OpBuilder> b;
+    std::unique_ptr<mlir::OpBuilder> builder;
```
Comment (Thirumalai-Shaktivel) [inline src/libasr/codegen/asr_to_mlir.cpp:44]: Minor:

@certik, I feel there is no need for builder to be unique_ptr, what do you think?

**Comment by Pranavchiku** [comment]
Always feel good to see your PRs :)

**Review by Pranavchiku — APPROVED** [review]

File: src/libasr/codegen/asr_to_mlir.cpp (hunk 1 above)
Comment (certik) [reply src/libasr/codegen/asr_to_mlir.cpp:44]: Probably it doesn't need to be `unique_ptr`.

**Comment by Thirumalai-Shaktivel** [comment]
Thanks for the reviews!

**Pull request merged**